dtmi/aaeon/hardwaremonitor-1.json

```diff
@@ -0,0 +1,101 @@
+{
+    "@id": "dtmi:AAEON:HardwareMonitor;1",
+    "@type": "Interface",
+    "displayName": "Hardware Monitor",
+    "@context": [
+        "dtmi:iotcentral:context;2",
```

Comment:
The central context is not needed since you are not using anything that require it. Consider to remove it.

Reply:
Hi @rido-min, thanks for your kindly reply. We will remove this later.
By the way, we've found this "dtmi:iotcentral:context;2" added automatically when users import their model file to IoT Central.
The full process likes that when we import our model file to Device Template then click edit DTDL, and we may find that the sentence mentioned above added automatically. Below is our screenshots:
<img width="785" alt="IoT Central Device Template Import Model" src="https://user-images.githubusercontent.com/60394729/131426571-b696e9e8-1dfe-48ac-ae11-bb460a4e2226.png">
<img width="367" alt="DTDL Update Automatically" src="https://user-images.githubusercontent.com/60394729/131426584-7ed8e127-ffb2-4f1d-b789-031349f6ea52.png">

Reply:
Yes, Central adds the context for it's internal usage, as well it adds localized elements to `displayName`. This is not a strict requirement, just a heads up that you model does not need the central context. If you prefer to keep it, it's ok. 

As I said in the initial comment only 1 requires some action, 2 and 3 are just suggestions.

Reply:
Thanks for your rapidly reply! We will remove it later, and we are just curious about this behavior.

Reply:
Ok, can you ask to @chiaotseng to review before approving?

Reply:
Sure. We are also waiting for @chiaotseng to review. But she looks like not in the office.

Reply:
@rido-min 
Hi Rido,
In fact, I helped them to submit models before, but the owner should be AAEON, not me.
They don't need to have my acknowledgment for submitting.
Thanks!!